tests/service/SyntaxTreeTests/SynTypeTests.fs

```diff
@@ -49,3 +49,65 @@ type T =
       ])) ->
         assertRange (3, 14) (3, 56) mTuple
     | _ -> Assert.Fail $"Could not get valid AST, got {parseResults}"
+
```

Comment:
Please let me know if you would like to have more tests.
I'm quite unfamiliar with this syntax, so I'd be happy to hear your proposals.